### DISCUSSION TITLE
Add a "within XX days" option for invoices due date

### DISCUSSION THREAD

**Discussion by flayks** [crater-invoice-inc/crater#582] [Ideas]
That would be a good option to be able to quickly select a period of time for the due date of an invoice, like on receipt, in 15/30/45 days or so, like what Wave is proposing.

This way we wouldn't have to calculate the days and would be able to quickly add +15 or +30 days.

![image](https://user-images.githubusercontent.com/273716/123076922-deabbd00-d419-11eb-9881-886d688e84d0.png)

**Comment by mohitpanjwani** [comment]
> That would be a good option to be able to quickly select a period of time for the due date of an invoice, like on receipt, in 15/30/45 days or so, like what Wave is proposing.
> 
> This way we wouldn't have to calculate the days and would be able to quickly add +15 or +30 days.
> 
> ![image](https://user-images.githubusercontent.com/273716/123076922-deabbd00-d419-11eb-9881-886d688e84d0.png)

What are your thoughts about adding a default option which allows you to set the days from settings, instead of having an option on individual invoices.

Do you often use different options for individual Invoices?

**Comment by flayks** [comment]
@mohitpanjwani I actually do as some customers have different payment terms like 30 or 45 days.
That would be good to have a default setting but also have the option for each invoice.

**Comment by mohitpanjwani** [comment]
> @mohitpanjwani I actually do as some customers have different payment terms like 30 or 45 days.
> That would be good to have a default setting but also have the option for each invoice.

Cool got it! Thanks for the feedback

**Comment by demamoh** [comment]
@mohitpanjwani  I would like to bring to attention another scenario, where there are multiple payment terms for a single invoice. 

For example: Invoice 1 
33% due 15 days after invoice date
67% due 60 days after invoice date

Basically a split milestone where a portion of the invoice comes due at different times for a single invoice. 

Would be happy to contribute code to do this (don't know too much much about how contribution works for this project).

**Comment by flayks** [comment]
Any progress on this idea? That would be very useful.

**Comment by ketandholakia** [comment]
Is it possible to add net days on customer records and when we make an invoice for that customer then the net day is taken from selected customer record and added as net days in invoice entry, which is editable if we want to change while making an invoice.
